mods/ctf/ctf_inventory/init.lua

```diff
@@ -12,21 +13,21 @@ local items = {
 	"   killing 2 people for every death.",
 	"",
 
-	minetest.colorize("#66a0ff", "Team Co-op"),
+	"#66a0ffTeam Co-op",
```

Comment:
I think that string concatenation like shown in the example below might improve readibility for a neglectible performance impact.

```suggestion
	"#66a0ff" .. "Team Co-op",
```

Reply:
I agree. Will add that.

Reply:
If you're keeping the colour separate, you might as well store it in a local var to keep things nice and clean.

Reply:
i.e.

```lua
local color = "#66A0FF"

-- *burp*

color .. "Team Co-op",

-- *Hachooo*

color .. "Player Rankings",

```